deno.jsonc

```diff
@@ -24,8 +24,8 @@
     }
   },
   "tasks": {
-    "test": "deno fmt --check && deno lint && deno test --allow-read --allow-net src",
-    "coverage": "deno test --allow-read --allow-net --coverage=.coverage src && deno coverage --exclude=fixtures --exclude=test --lcov --output=lcov.info .coverage && deno run --allow-read https://deno.land/x/code_coverage@0.2.0/cli.ts"
+    "test": "deno fmt --check && deno lint && deno test --allow-read --allow-net --allow-write --allow-run src",
```

Comment:
These new permissions are now required in this repo due to the deno-slack-builder code being moved in here. -builder requires reading and writing files (to read app manifest, and write out app bundles).